app/src/main/java/com/sun/heartrate/ui/guideline/GuidelineFragment.kt

```diff
@@ -17,5 +17,5 @@ class GuidelineFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater,
```

Comment:
Format code theo dạng này nhé.
```suggestion
    override fun onCreateView(
        inflater: LayoutInflater,
        ...
    ): View? = ...
```